WordPress/src/main/res/layout/plans_list_item.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white"
+    android:clickable="true"
+    android:focusable="true"
+    android:minHeight="?attr/listPreferredItemHeight"
+    tools:layout_height="?attr/listPreferredItemHeightLarge">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="?attr/selectableItemBackground">
+
+        <ImageView
+            android:id="@+id/plan_image"
+            android:layout_width="@dimen/activity_log_icon_size"
+            android:layout_height="@dimen/activity_log_icon_size"
+            android:layout_alignParentStart="true"
+            android:layout_centerVertical="true"
+            android:layout_margin="@dimen/activity_log_icon_margin"
+            android:contentDescription="@string/description_user"
+            android:src="@drawable/ic_plans_white_24dp"
+            android:tint="@color/grey">
```

Comment:
```suggestion
            tools:tint="@color/grey"/>
```

The `android:tint` attribute causes the plan icons to be `grey` (`#87a6bc`) even after they have been loaded form the icon URL.  See the screenshot below for illustration.

<img src="https://user-images.githubusercontent.com/3827611/52675478-31891780-2ee4-11e9-8299-0e2f117ed01d.png" width="240">

We should change the namespace to `tools`.